### DISCUSSION TITLE
How can I update brunch to the newest verion ?

### DISCUSSION THREAD

**Discussion by MatrixXD** [sebanc/brunch#2150] [Show and tell]
How can I update ?

**Comment by ninetsf96** [comment]
Use brcr-update or use the Brunch PWA enabling the pwa option in Brunch Settings, or for manual update, try this enabling the sudo for crosh and enable-updates options in Brunch Settings:
sudo chromeos-update -f ~/MyFiles/Downloads/yourbrunchfile.tar.gz
Change "yourbrunchfile" to the correct name of your file
